flash/core/data/data_module.py

```diff
@@ -786,6 +786,112 @@ def from_tensors(
             **preprocess_kwargs,
         )
 
+    @classmethod
+    def from_data_sequence(
```

Comment:
Perhaps we could have a `graph/data.py` with a `GraphDataModule` that has this since it is unique to graph stuff?

Reply:
Are you sure this is unique? I am thinking of any DataList. I remember there used to be one in fastai. The idea is to have
DataList = [Data_object, ..., Data_object] where Data_object depends on the task

Reply:
Perhaps I am outdated though and this way of working is no longer common?

Reply:
@PabloAMC Ok, I think we can do that, I guess we already have the `SequenceDataSource` so this is a good fit. I only mean unique in the sense that we don't use it for other tasks, but we could do :smiley: